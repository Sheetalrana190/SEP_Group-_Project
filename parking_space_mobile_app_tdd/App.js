import React from 'react';
import { NavigationContainer } from '@react-navigation/native';
import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
import HomeScreen from './screens/HomeScreen';
import PostScreen from './screens/PostScreen';
import BookingScreen from './screens/BookingScreen';

const Tab = createBottomTabNavigator();

export default function App() {
  return (
    <NavigationContainer>
      <Tab.Navigator testID="tab-bar">
        <Tab.Screen name="Home" component={HomeScreen} />
        <Tab.Screen name="Post" component={PostScreen} />
        <Tab.Screen name="Bookings" component={BookingScreen} />
      </Tab.Navigator>
    </NavigationContainer>
  );
}
