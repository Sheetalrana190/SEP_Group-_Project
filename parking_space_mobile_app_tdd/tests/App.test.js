import React from 'react';
import { render } from '@testing-library/react-native';
import App from '../App';

describe('App Navigation', () => {
  it('renders the Home screen by default', () => {
    const screen = render(<App />);
    expect(screen.getByText('Lot A')).toBeTruthy(); // or 'Lot B', 'Lot C'
  });

  it('renders successfully', () => {
    const screen = render(<App />);
    expect(screen).toBeDefined();
  });
});