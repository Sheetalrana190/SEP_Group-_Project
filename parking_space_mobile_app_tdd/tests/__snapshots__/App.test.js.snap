// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`App Navigation matches snapshot 1`] = `null`;
